website/src/components/communitySpotlightCard/index.js

```diff
@@ -100,7 +100,7 @@ function CommunitySpotlightCard({ frontMatter, isSpotlightMember = false }) {
           </div>
         )}
         {description && !isSpotlightMember && (
-          <p className={styles.spotlightMemberDescription}>{truncateText(description)}</p>
+          <p className={styles.spotlightMemberDescription} dangerouslySetInnerHTML={{__html: truncateText(description)}} />
```

Comment:
Ruh-roh, `dangerouslySetInnerHtml`! 😆 I'm sure it's fine, but just for my own curiosity: what's this change for?

Reply:
This is to support html links within the `description` field on the cards within the spotlight index page. Doing the same for the description on individual spotlight member pages.